src/Features/VisualBasic/Portable/InlineHints/VisualBasicInlineParameterNameHintsService.vb

```diff
@@ -96,5 +96,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.InlineHints
             Dim propertySymbol = TryCast(parameter.ContainingSymbol, IPropertySymbol)
             Return propertySymbol IsNot Nothing AndAlso propertySymbol.IsDefault
         End Function
+
+        Protected Overrides Function HintText(parameterName As String) As String
+            Return parameterName & ":= "
```

Comment:
📝 VB typically doesn't format with a space around `:=`, so we should consider omitting this one if it doesn't mess up the rendering

Reply:
Yeah, without the space, the argument is right up against the adornment. Which I'm sure can also be fixed by adding some padding to the adornment.

Reply:
can you show pictures of each?

Reply:
VB:
<img width="1060" alt="image" src="https://user-images.githubusercontent.com/40616383/143929332-8f2397e9-227b-4eca-a018-2b74c97c7821.png">

C#:
<img width="396" alt="image" src="https://user-images.githubusercontent.com/40616383/143929278-e302c943-fb17-45e3-adec-fc6e61f33a46.png">